webapp/components/display-logo.html

```diff
@@ -114,9 +116,8 @@
         if (!showSource || !product.labels) {
           return '';
         }
-        const sources = ['taskcluster', 'buildbot', 'msedge'];
-        const labels = new Set(product.labels);
-        return sources.find(s => labels.has(s));
+        const sources = window.wpt.Sources;
+        return product.labels.find(s => sources.has(s));
```

Comment:
Nit: maybe just combine the two lines.

Reply:
I think that would execute the getter, which creates a new set, every loop.